task/06-conditions-n-loops-tasks.js

```diff
@@ -29,8 +29,16 @@
  *  21 => 'Fizz'
  *
  */
-function getFizzBuzz(num) {
-    throw new Error('Not implemented');
+function getFizzBuzz(num) {    
+
+    if (num % 3 == 0 && num % 5 == 0) return "FizzBuzz";
+    else if (num % 3 == 0) return "Fizz";
+    else if (num % 5 == 0) return "Buzz";
+    else return num;
+
+    // return (num % 3 == 0 && num % 5 == 0) ? "FizzBuzz" : 
+    //         (num % 3 == 0) ? "Fizz" : 
```

Comment:
Everything is right, but please try to make sure that there are no commented-out lines left in the code.

Reply:
Okay, Thanks a lot. Sometimes I left (I mean the other tasks 7, 8, 9) the commented lines as the alternative way of solution. But got it! It's better not to do like this. Thanks!

Reply:
Your second solution is not so preferred here because of the deep nesting. Try to use a ternary expression if you need to set only 1 condition (only one 'if').
Thank you too!

Reply:
> Your second solution is not so preferred here because of the deep nesting. Try to use a ternary expression if you need to set only 1 condition (only one 'if').
> Thank you too!

I corrected the mistakes, thanks. Will the PR be closed?